io.github.obsidian-level-maker.yml

```diff
@@ -0,0 +1,44 @@
+app-id: io.github.obsidian-level-maker
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: obsidian.sh
+finish-args:
+- --socket=wayland
+- --socket=fallback-x11
+- --socket=x11
+- --filesystem=home:rw
+modules:
+  - name: obsidian
+    buildsystem: cmake
+    config-opts:
+    - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: git
+        url: http://github.com/obsidian-level-maker/Obsidian
+        tag: Obsidian-v20-20230509
+        commit: fb2c9f6269ced76643fa4c362738784ebae26cf7
+    build-commands:
+      - install -D obsidian /app/bin/obsidian
+  - name: startscript
+    buildsystem: simple
+    sources:
+      - type: script
+        commands:
+          - obsidian --install /app
```

Comment:
not sure what `--install` does, but if it is supposed to perform an install, then this will fail as `/app` is read only.

Reply:
It designates the /app directory as the location of various files that it reads on startup. It does not "install" anything, despite the parameter name.